modules/osm/multipolygon.js

```diff
@@ -210,7 +210,6 @@ export function osmJoinWays(toJoin, graph) {
             }
 
             if (!nodes) {     // couldn't find a joinable way/member
-                doneSequence = true;
```

Comment:
Does this change the functionality? If not, can we remove `doneSequence` entirely?

Reply:
I think it's needed because of the `break` below to exit from the loop.